addons/hr_holidays/models/hr_leave.py

```diff
@@ -260,6 +262,34 @@ def _auto_init(self):
                            self._table, ['date_to', 'date_from'])
         return res
 
+    @api.depends_context('uid')
+    def _compute_description(self):
+        self.check_access_rights('read')
+        self.check_access_rule('read')
+
+        is_officer = self.user_has_groups('hr_holidays.group_hr_holidays_user')
+        user_id = self.env.user.id
+
+        for leave in self:
+            if is_officer or leave.user_id.id == user_id or (leave.manager_id and leave.manager_id.id == user_id):
```

Comment:
Last part of the if clause can be simplified
```python
if is_officer or leave.user_id.id == user_id or leave.manager_id.id == user_id:
```

Also no need to compare ids instead of directly the recordsets.